tests/unit/proxy-api/proxy-api.service.spec.ts

```diff
@@ -0,0 +1,112 @@
+import { Test, TestingModule } from '@nestjs/testing';
+// import { ApiService } from '../api.service';
```

Comment:
Globally it looks good to me, I see 4 code smells: 1 commented code and 3 unused import.
Do you thing it could be cleaned before merging? https://sonarcloud.io/project/issues?id=Sanofi-IADC_konviw&open=AX5VPK0A69POjsP1H3Qm&pullRequest=286&resolved=false&types=CODE_SMELL

Reply:
Thanks for noticing this, yes I'll remove this one for the api.service (copy/paste error from another test I think).

I'll check the other ones too